meson.build

```
@@ -535,7 +535,7 @@ if build_tools
         wayland_protocols_dep = dependency('wayland-protocols', version: '>=1.12', required: false)
         wayland_scanner_dep = dependency('wayland-scanner', required: false, native: true)
         if not wayland_client_dep.found() or not wayland_protocols_dep.found() or not wayland_scanner_dep.found()
-            error('''The Wayland xkbcli programs require wayland-client >= 1.2.0, wayland-protocols >= 1.7 which were not found.
+            error('''The Wayland xkbcli programs require wayland-client >= 1.2.0, wayland-protocols >= 1.12 which were not found.
```

```suggestion
            error('''The Wayland xkbcli programs require wayland-client and wayland-protocols which were not found.
```
I'd argue to drop the versions here because meson will print something like this anyway:
```
Dependency wayland-protocols found: NO found 1.11 but need: '>=1.12'
```